### ISSUE TITLE
3.10.0 AppDeployToolkitConfig.xml Typo?

### ISSUE THREAD

**Issue by alexlarner** [PSAppDeployToolkit/PSAppDeployToolkit#915]
Is `&amp;` supposed to be in the values for the `ClosePrompt_Button[Continue|Close|Defer]` properties in the AppDeployToolkitConfig.xml? It's not there for the majority of the localizations and it's not in any of the localizations for 3.9.3.

**Comment by sintaxasn** [comment]
Yes - it was added to German and English. It adds a shortcut key for each button. Hold down Alt key, and you'll see the relevant keys get hightlighted. **D**efer, Close **P**rograms and **C**ontinue

**Comment by sintaxasn** [comment]
Phew! You scared me there with the issue title coming in minutes after release :O
